dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -54,4 +54,10 @@ tcp.hostNameIncludeList=Host name include list
 tcp.hostNameIncludeList.desc=A comma-separated list of host names that are allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid. 
 
 tcp.hostNameExcludeList=Host name exclude list
-tcp.hostNameExcludeList.desc=A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid.
\ No newline at end of file
+tcp.hostNameExcludeList.desc=A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid.
```

Comment:
Suggestion:
A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid; however, *.abc.* is not valid.

Reply:
This tcpOption is not being added to the documentation, it already documented. Git just picked up a line break change from the original file.